release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,152 @@
+# 3.10.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supporting remote first-time login](#supporting-remote-first-time-login)
+- [Supporting linked docs](#supporting-linked-docs)
+- [Privacy Policies](#privacy-policies)
+- [More granular replication depth configuration for offline users](#more-granular-replication-depth-configuration-for-offline-users)
+- [Improved geolocation data saved for reports](#improved-geolocation-data-saved-for-reports)
+- [Improved performance of processing deleted docs](#improved-performance-of-processing-deleted-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes 
+
+### Breaking changes
+
+#### Updates to meta databases replication
+
+The replication of users meta databases to the conglomerate, `medic-users-meta` database, is no longer configurable. 
+This task now runs every day, at 2am UTC, and replicates `feedback` and `telemetry` documents to `medic-users-meta` database.
+
+#### Android wrapper update required for new features
+
```

Comment:
Should we mention GPS improvement as well since it requires a new wrapper at time of upgrade to be functional?

Reply:
The cht-core side GPS is compatible with previous versions of the android wrapper.